extensions/tika/deployment/src/main/java/io/quarkus/tika/deployment/ParsersConfigBuildItem.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.tika.deployment;
+
+import java.util.List;
+import java.util.Map;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+import io.quarkus.tika.runtime.TikaParserParameter;
+
+public final class ParsersConfigBuildItem extends SimpleBuildItem {
```

Comment:
Any change we could name it `TikaParsersConfigBuildItem`?

Reply:
Done